enterprise/lang-js/src/test/java/io/crate/operation/language/JavascriptUserDefinedFunctionTest.java

```diff
@@ -116,7 +117,24 @@ public void testInvalidJavascript() throws ScriptException{
         );
 
         String validation = udfService.getLanguage(JS).validate(udfMeta);
-        assertThat(validation, startsWith("Invalid JavaScript in function 'doc.f(double)': <eval>:1:27"));
+        assertThat(validation, startsWith("Invalid JavaScript in function 'doc.f(double)'"));
+        assertThat(validation, endsWith("^ in <eval> at line number 1 at column number 27"));
+    }
+
+    @Test
+    public void testInvalidJavascriptThrowsAssertionError() {
```

Comment:
If I may join in on the naming discussion, I think something along the lines of `test_validate_catches_assertion_errors` or `test_validate_never_raises` would be more accurate.

Reply:
👍 

Would rephrase to `testJavascriptExceptionsAreCaught`